Analyze dataframe and keep it until the next analysis (#3497)
freqtrade/freqtrade: docs/bot-basics.md

# Freqtrade basics

> This page will try to teach you some basic concepts on how freqtrade works and operates.

Reviewer: ```suggestion
This page provides you some basic concepts on how Freqtrade works and operates.
```

## Freqtrade terminology

> * Trade: Open position

Reviewer: ```suggestion
* Trade: Open position.
```
